matlab/test/Ice/slicing/exceptions/ClientPrivate.ice

```diff
@@ -49,7 +49,6 @@ class BaseClass
     string bc;
 }
 
-["format:sliced"]
```

Comment:
Even through the "class fixes" have not been applied to MATLAB yet, `format:sliced` doesn't matter for user exceptions in MATLAB: MATLAB is client-only and doesn't marshal user exceptions.